website/static/js/folderPicker.js

```diff
@@ -155,7 +155,7 @@
                 }
                 tb.options.folderArray = folderArray;
             }
-
+            console.log(tb.treeData);
```

Comment:
I will remove this when I merge, but please don't check in `console.log` statements.